lib/stream.js

```diff
@@ -33,6 +35,9 @@ Stream.Duplex = require('_stream_duplex');
 Stream.Transform = require('_stream_transform');
 Stream.PassThrough = require('_stream_passthrough');
 
+Stream.pipeline = pump;
+Stream.onEnd = eos;
```

Comment:
`onEnd` is a confusing name in the context of streams, we should use something else

Reply:
Open for suggestions :)

Reply:
Can we used `finished`, like in `mississippi`?